### ISSUE TITLE
HS430

### ISSUE THREAD

**Issue by rotaflex** [steveredden/homebridge-kasa-lightstrip#5]
This new model light strip couldn't be controlled via HomeKit

**Comment by JBruster1119** [comment]
Followed the pre-requisites and still not getting it to connect with home kit

**Comment by JBryant915** [comment]
Yes, newer version of python-Kasa (0.4.0) requires --type light strip as the argument in the command structure as the plugin looks to be trying to pass --lightstrip only which looks to work with python-kasa (0.4.0). Unknowingly, I installed the new version and it wouldn't work so I went poking around and found that it was rejecting the command through cli referencing the --lightstrip argument. I read another posted issue here and installed 0.4.0 and it will pass the command from command line but not through the plugin. End results, it still doesn't work in homekit... I've added to the issue I referenced in hopes of getting a resolution. The other TPlink plugin isn't working for me with the KL430 either.
